ui/app/templates/allocations/allocation/task/index.hbs

```diff
@@ -44,9 +44,19 @@
       >
         {{this.model.state}}
       </span>
-    </div>
-    <div>
+    </PH.Title>
+    <PH.Actions>
       {{#if this.model.isRunning}}
+
+        {{#if this.shouldShowActions}}
+          <Hds::Dropdown class="actions-dropdown" as |dd|>
+            <dd.ToggleButton @color="secondary" @text="Actions" @size="medium" />
+            {{#each this.model.task.actions as |action|}}
+              <dd.Interactive {{on "click" (perform this.runAction action (get (object-at 0 action.allocations) "id"))}} @text="{{action.name}}" />
+            {{/each}}
+          </Hds::Dropdown>
+        {{/if}}
```

Comment:
The spacing between the buttons at different levels is a bit jarring. 

I think we're also missing the `Actions` button at the alloc and group level?

<img width="1218" alt="image" src="https://github.com/hashicorp/nomad/assets/775380/2723b7fd-5f93-4973-a259-ad99891a1463">
<img width="1216" alt="image" src="https://github.com/hashicorp/nomad/assets/775380/f0f059ca-4bca-4aeb-a8f7-3b4a1e39f9c2">
<img width="1206" alt="image" src="https://github.com/hashicorp/nomad/assets/775380/d0330e53-8116-4725-8a53-74ce697e08c1">

Reply:
The spacing on the Task and Job page will be the norm for all pages; I'm rolling it the [new PageHeader component](https://helios.hashicorp.design/components/page-header) on a page-by-page basis.

I hadn't considered the spacing issue too off-putting, but this comment makes me think should do a wholesale switch-over soon.

Reply:
Actions on Group and Alloc level, I was planning to leave until the Actions service from https://github.com/hashicorp/nomad/pull/19084 is wrapped up; I'll handle that in a future PR.